### ISSUE TITLE
Unable to create value classes

### ISSUE THREAD

**Issue by sauerburger** [almond-sh/almond#799]
I ran into the following issue while creating a value class in an almond-based Juptyer notebook:

```scala
class Kelvin(val t: Double) extends AnyVal
```
```scala
cmd0.sc:7: not found: value res0
  .declareVariable("res0", res0); Iterator() }, 
                           ^cmd0.sc:1: value class may not be a member of another class
class Kelvin(val t: Double) extends AnyVal
      ^Compilation Failed

Compilation Failed
```

I can reproduce the error via the "Try it online" button on https://almond.sh/ for Scala 2.11, 2.12 and 2.13 (although the error message is less verbose in 2.11). I assume, almond silently injects another base class which is not allow for the value class. Is there a way to solve this?

**Comment by alexarchambault** [comment]
@sauerburger It's a known limitation of the way almond "wraps" the user code (basically, the code is passed to the compiler like `class Wrapper { …user code… }`, which plays well with Java serialization, for spark, but prevents value classes or macros to be defined as is.)

As a workaround, explicitly putting your code in an `object` should disable class wrapping for the current cell, and should compile fine:
```scala
object Definitions {
  class Kelvin(val t: Double) extends AnyVal
}
```

**Comment by sauerburger** [comment]
Thanks, the workaround solve the issue.

**Comment by Quafadas** [comment]
Interestingly, for me this workaround doesn't appear to work... same error - @sauerburger do you mind telling which versions are you using?

**Comment by sauerburger** [comment]
@Quafadas I've tested the workaround with my local setup.
```
Almond 0.6.0
Ammonite 1.6.7
Scala library version 2.11.12 -- Copyright 2002-2017, LAMP/EPFL
Java 1.8.0_262
```

With the [Try it online](https://mybinder.org/v2/gh/almond-sh/examples/master?urlpath=lab%2Ftree%2Fnotebooks%2Findex.ipynb) button, the workaround succeeds only for Scala 2.11. With 2.12 and 2.13, I get

```
cmd0.sc:9: not found: value res0
  .declareVariable("res0", res0); Iterator() }, 
                           ^Compilation Failed

Compilation Failed
```

**Comment by Quafadas** [comment]
Interesting; 
Almond 0.11.1
Ammonite 2.3.8-36-1cce53f3
Scala library version 2.13.4 -- Copyright 2002-2020, LAMP/EPFL and Lightbend, Inc.
Java 1.8.0_265

With these versions, this code works; 
```
case class ATest(i:Int, str:String)
ATest(2, "Simon")

defined class ATest
res1_1: ATest = ATest(i = 2, str = "Simon")
```

Unless I have the variable inspector installed... in which case I get the error message you recieved above. I'll double check and probably opena a bug against the variable inspector...

**Comment by Quafadas** [comment]
Thanks for the feedback.

**Comment by alexarchambault** [comment]
@Quafadas @sauerburger The upcoming version `0.11.2` should fix the error in 2.12 and 2.13 in the binder demo, or when the variable inspector is enabled, thanks to https://github.com/almond-sh/almond/pull/793.

**Comment by Quafadas** [comment]
Oh - cool, thanks!
